airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.py

```diff
@@ -0,0 +1,66 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Example DAG demonstrating the usage of the BashOperator."""
+
+from datetime import timedelta
+
+from airflow import DAG
+from airflow.providers.airbyte.operators.airbyte import AirbyteTriggerSyncOperator
+from airflow.providers.airbyte.sensors.airbyte import AirbyteJobSensor
+from airflow.utils.dates import days_ago
+
+args = {
+    'owner': 'airflow',
+}
+
+with DAG(
+    dag_id='example_airbyte_operator',
+    default_args=args,
+    schedule_interval=None,
+    start_date=days_ago(1),
+    dagrun_timeout=timedelta(minutes=60),
+    tags=['example'],
+) as dag:
+
+    # [START howto_operator_airbyte_synchronous]
+    sync_source_destination = AirbyteTriggerSyncOperator(
+        task_id='airbyte_sync_source_dest_example',
+        airbyte_conn_id='airbyte_default',
+        connection_id='15bc3800-82e4-48c3-a32d-620661273f28',
+    )
+    # [END howto_operator_airbyte_synchronous]
+
+    # [START howto_operator_airbyte_asynchronous]
+    async_source_destination = AirbyteTriggerSyncOperator(
+        task_id='airbyte_async_source_dest_example',
+        airbyte_conn_id='airbyte_default',
+        connection_id='15bc3800-82e4-48c3-a32d-620661273f28',
+        asynchronous=True,
+    )
+
+    airbyte_sensor = AirbyteJobSensor(
+        task_id='airbyte_sensor_source_dest_example',
+        airbyte_job_id=async_source_destination.output,
```

Comment:
@dstandish  and @tuanchris The CI broke in **Backport packages: wheel** step in this line. Complained that the `AirbyteTriggerSyncOperator` doesn't have `output` property. Any suggestions?
I could use the `task_instance xcom_pull`.
```
        dataproc_job_id="{{task_instance.xcom_pull(task_ids='spark_task_async')}}",
```

Reply:
maybe you meant to tag @turbaszek

Reply:
> I could use the task_instance xcom_pull

yeah that `output` thing is new... i think it may be only when you use `TaskFlow` api

i cannot look more deeply until possibly this evening

Reply:
No problem @dstandish. I'd tested in a local setup the example_dag and ran without problem using the `output` with Airflow 2.0, I want to check with you how to proceed with this CI conflict.

Reply:
Ok forgive me, XcomArg is new to me.  So is backports process.

I found this line which suggests that for "legacy" operators (which I presume to mean < 2.0?) we must use jinja to use xcomargs. 

https://github.com/apache/airflow/blob/master/airflow/models/xcom_arg.py#L46

It looks like backports assume 1.10.x version of BaseOperator.   So what this suggests is that you need to change your example to use jinja.

@potiuk may have some guidance as to the best thing to do here.  Perhaps examples should always be written in backport-compatible way.  Or perhaps there can be versioned example files.

Reply:
Yes, in example DAGs we still use old approach to enable backporting. The XCromArgs do not work with Airflow 1.10.X

Reply:
I changed to support the backporting.